src/ports/postgres/modules/svm/svm.py_in

```diff
@@ -676,27 +985,32 @@ def svm_help(schema_madlib, message, is_svc, **kwargs):
 def svm(schema_madlib, source_table, model_table,
         dependent_varname, independent_varname, kernel_func,
         kernel_params, grouping_col, params, is_svc,
-        verbose, detect_novelty=False, **kwargs):
+        verbose, add_intercept=False, **kwargs):
```

Comment:
`fit_intercept` has been added to `kernel_params`. We can still keep the `array[1, ...]` usage for non-kernel case since we anyways need that in other modules. The add_intercept here is only to override the user-provided param. I'll change this to use `add_intercept=None` and add a note to make it clearer.